src/components/CreateBoxe/CreateBoxe.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+
+function CreateBoxe() {
+  const [value, setValue] = useState('red');
+
+  function handleChange(event) {
+    setValue(event.target.value);
+  }
+
+  return (
+    <div>
+      <form action="" method="get" className="">
+        <div className="">
+          <label htmlFor="nameOfBoxes">Nom du coffret : </label>
+          <input type="text" name="name" id="name" required />
+        </div>
+        <div className="">
+          <label htmlFor="nameOfBottle">Nom de la bouteille : </label>
+          <input type="text" name="name" id="name" required />
+        </div>
+        <div>
+          <label htmlFor="TypeOfBottle">
+            Categorie de la bouteille :
+            <select value={value} onChange={(event) => handleChange(event)}>
+              <option value="red"> rouge </option>
+              <option value="white"> blanc </option>
+              <option value="sparkling"> pétillant </option>
+            </select>
+          </label>
+          <br />
```

Comment:
`<br>` should only be used inside paragraphes: remove it. use css to add spacing